### PR TITLE
Replace 4.6 references with 4.7

### PR DESCRIPTION
**Pull request by fedepaol**
Master now points to 4.7, all the references must be aligned.

### CODE REVIEW

File: cnf-tests/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ COPY . .
 RUN make test-bin
 RUN git rev-list -1 HEAD > ./cnf-tests/bin/cnftests-sha.txt
 
-FROM quay.io/openshift/origin-oc-rpms:4.6 AS oc
+FROM quay.io/openshift/origin-oc-rpms:4.7 AS oc
```
Comment (fromanirh) [inline cnf-tests/Dockerfile:7]: looks good, but did you check we have these images available already? (I'd not just assume that)

File: external-tests/origintests/test.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
-OSETESTS_IMAGE="${OSETESTS_IMAGE:-quay.io/openshift/origin-tests:4.6}"
+OSETESTS_IMAGE="${OSETESTS_IMAGE:-quay.io/openshift/origin-tests:4.7}"
```
Comment (fromanirh) [inline external-tests/origintests/test.sh:11]: same question as above

**Review by fromanirh — COMMENTED** [review]
looks good, but please check the inline comments

File: external-tests/origintests/test.sh (hunk 2 above)
Comment (fedepaol) [reply external-tests/origintests/test.sh:11]: CI will tell I guess :-)

File: cnf-tests/Dockerfile (hunk 1 above)
Comment (fedepaol) [reply cnf-tests/Dockerfile:7]: Let's wait for CI to tell.
I hope the mirroring jobs are already in place

**Comment by fedepaol** [comment]
@fromanirh images lane passed, meaning the dockerfiles are good

File: cnf-tests/Dockerfile (hunk 1 above)
Comment (fromanirh) [reply cnf-tests/Dockerfile:7]: works for me

**Pull request merged**